### ISSUE TITLE
out of vertex space.

### ISSUE THREAD

**Issue by lyd0527** [bmansfeld/QTLseqR#9]
```
> df_filt <- runGprimeAnalysis(
+     SNPset = df_filt,
+     windowSize = 1e6,
+     outlierFilter = "deltaSNP")
Counting SNPs in each window...
Calculating tricube smoothed delta SNP index...
Error in mutate_impl(.data, dots) : 
  Evaluation error: newsplit: out of vertex space.
```

**Comment by bmansfeld** [comment]
Hi,
This error comes from the locfit package running out of memory for calculating the smoothing function. 
See here: https://stat.ethz.ch/pipermail/r-help/2003-March/031384.html
Increase your window size and it should go away. 
-Ben

**Comment by bmansfeld** [comment]
Ok so i've added the ability to pass `maxk` parameter (through `...`) to the `locfit` function in both `runQTLseqAnalysis` and `runGprimeAnalysis` functions. 
I've also updated the documentation for the functions and the vignette with a little explanation. All is available in version 0.7.3, out now.
This should make it easier for users to address this. I warn however that if you are getting this error you should probably increase your window size rather than increase the memory allocation. The point of the analysis is to smooth noise and if your windows are too small this won't happen.
Thanks for the feedback.
Ben

**Comment by ghost** [comment]
Hi Ben,
Thus I'm a green hand in R programming, could you please tell me which command line could be a possible solution of this problem, such as change the maxk parameter, because I believe that I have chosen a proper window size.
Thanks!
